stratum/hal/lib/barefoot/macros.h

```diff
@@ -72,11 +72,11 @@ class BooleanBfStatus {
 
 // A macro for simplify checking and logging the return value of a SDE function
 // call.
-#define RETURN_IF_BFRT_ERROR(expr)                            \
-  if (const BooleanBfStatus __ret = BooleanBfStatus(expr)) {  \
-  } else /* NOLINT */                                         \
-    return MAKE_ERROR(__ret.error_code())                     \
-           << "'" << #expr << "' failed with error message: " \
+#define RETURN_IF_BFRT_ERROR(expr)                                            \
+  if (const BooleanBfStatus __ret = BooleanBfStatus(expr)) {                  \
+  } else /* NOLINT */                                                         \
+    return MAKE_ERROR(__ret.error_code()).severity(WARNING).without_logging() \
```

Comment:
This needs to be fixed once we decided what to pursue. Either set the level or disable it.
Currently both is done to document the options.